### PR TITLE
Add config file handling in app

### PR DESCRIPTION
**Pull request by amitschang**
Read config file on startup and write upon reconfiguration. ~For the moment the location and whether to write are non-configurable.~

Resolves https://github.com/ssec-jhu/evolver-ng/issues/24

Also sneaks in a change to set host and port for http server via environment variables

### CODE REVIEW

File: evolver/app/main.py [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+import yaml
 from fastapi import FastAPI
+from pathlib import Path
 
 from evolver.base import require_all_fields
 from evolver.device import Evolver, EvolverConfig
 from .. import __project__, __version__
 
 
+EVOLVER_CONFIG_FILE=Path('evolver.yml')  # in current directory
```
[the remark this answers is not in the document]
Comment (amitschang) [reply evolver/app/main.py]: thanks, totally missed that!

**Review by jamienoss — CHANGES_REQUESTED** [review]
Hey, since I was the one that suggested the revisions before, I felt bad to do so yet again, so I went ahead and made them for us over in #76.

**Comment by jamienoss** [comment]
@amitschang Also, I think the files should rep descriptors not just the config such that the file also has the classinfo so that when looking at a config file you know what model it's for.

**Review by jamienoss — APPROVED** [review]

**Comment by jamienoss** [comment]
@amitschang My approval doesn't count since I last contributed - I would just admin merge this.

**Pull request merged**